packages/lb-components/src/components/pointCloudView/PointCloud2DView.tsx

```diff
@@ -7,27 +7,49 @@ import { connect } from 'react-redux';
 import { pointCloudLidar2image, cKeyCode } from '@labelbee/lb-annotation';
 import { LabelBeeContext } from '@/store/ctx';
 import { a2MapStateToProps, IA2MapStateProps } from '@/store/annotation/map';
-import { toolStyleConverter } from '@labelbee/lb-utils';
+import { IPointCloudBox, IPolygonPoint, toolStyleConverter } from '@labelbee/lb-utils';
 import PointCloud2DSingleView from './PointCloud2DSingleView';
 import TitleButton from './components/TitleButton';
 import { LeftOutlined } from '@ant-design/icons';
 import classNames from 'classnames';
 import EscSvg from '@/assets/annotation/common/icon_esc.svg';
 import LeftSquareOutlined from '@/assets/annotation/common/icon_left_squareOutlined.svg';
 import RightSquareOutlined from '@/assets/annotation/common/icon_right_squareOutlined.svg';
+import { IMappingImg } from '@/types/data';
 
 // TODO, It will be deleted when the exported type of lb-annotation is work.
-interface IAnnotationDataTemporarily {
+export interface IAnnotationDataTemporarily {
   type: string;
-  annotation: any;
+  annotation: {
+    id: number | string;
+    pointList: IPolygonPoint[];
+    color: string;
+    stroke: string;
+    fill: string;
+  };
+}
+
+interface ITransferViewData {
+  type: string;
+  pointList: {
+    id: string;
+    x: number;
+    y: number;
+  }[];
+}
+
+export interface IAnnotationData2dList {
```

Comment:
Don't use `List`. Because it just an  `Object`.

And you can rename this. because of redefining this 2dview to show all camera.